ibis/impala/compat.py

```diff
@@ -14,4 +14,5 @@
 
 from impala.error import Error as ImpylaError  # noqa
 from impala.error import HiveServer2Error as HS2Error  # noqa
+from impala._thrift_api import TGetOperationStatusReq, TOperationState
```

Comment:
For other reviewers who might do a double-take at the unused import, this is so these can be imported by other things that are importing `ibis.impala.compat`